### PR TITLE
Fix install for iOS simulator, and add ability to uninstall

### PR DESCRIPTION
**Pull request by tvolkert**
#4216

### CODE REVIEW

**Comment by tvolkert** [comment]
@devoncarew @chinmaygarde

**Comment by abarth** [comment]
`[hint] The value of the local variable 'stdout' is not used (packages/flutter_tools/lib/src/ios/devices.dart, line 148, col 14)`

**Comment by tvolkert** [comment]
@abarth done

File: packages/flutter_tools/lib/src/ios/simulators.dart [hunk 1]
```diff
@@ -361,6 +365,17 @@ class IOSSimulator extends Device {
   }
 
   @override
+  bool isAppInstalled(ApplicationPackage app) {
+    return exitsHappy(<String>[
+      'xcrun',
+      'simctl',
+      'get_app_container',
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/ios/simulators.dart]: You could add this as a new method to `SimControl`; that class already wraps a lot of `simctl` commands.

File: packages/flutter_tools/lib/src/commands/install.dart [hunk 2]
```diff
@@ -34,8 +34,11 @@ bool installApp(Device device, ApplicationPackage package) {
   if (package == null)
     return false;
 
-  if (device.isAppInstalled(package))
-    return true;
+  if (device.isAppInstalled(package)) {
+    printStatus('Uninstalling old version...');
+    if (!device.uninstallApp(package))
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/commands/install.dart]: Do we want to fail the install if we can't uninstall the old version?

**Comment by devoncarew** [comment]
lgtm

**Pull request merged**